storagemarket/impl/providerstates/provider_states.go

```diff
@@ -208,6 +210,23 @@ func DecideOnProposal(ctx fsm.Context, environment ProviderDealEnvironment, deal
 	return ctx.Trigger(storagemarket.ProviderEventDataRequested)
 }
 
+const restartTimeout = 1 * time.Hour
```

Comment:
Is `restartTimeout` used anywhere?

Reply:
nope accidental leftover -- became defaultRestartTimeout -- will delete